src/vs/workbench/contrib/debug/browser/debug.contribution.ts

```diff
@@ -439,6 +439,11 @@ configurationRegistry.registerConfiguration({
 			description: nls.localize({ comment: ['This is the description for a setting'], key: 'openExplorerOnEnd' }, "Automatically open the explorer view at the end of a debug session."),
 			default: false
 		},
+		'debug.closeReadonlyTabsOnEnd': {
+			type: 'boolean',
+			description: nls.localize({ comment: ['This is the description for a setting'], key: 'closeReadonlyTabsOnEnd' }, "Automatically close the read-only tabs at the end of a debug session."),
```

Comment:
I'm not a big fan of this wording, because read-only tabs can come from other places, but I'm not sure how to describe it in a better way without getting superfluous